maintainers/maintainer-list.nix

```diff
@@ -5112,7 +5112,7 @@
   };
   tobim = {
     email = "nix@tobim.fastmail.fm";
-    github = "tobimpub";
+    github = "tobim";
```

Comment:
As per brand new policy if you set `github` you must also set `githubid`. Your `githubid` is [858790](https://api.github.com/users/tobim);

Reply:
This is already fixed in master, I'll do a rebase.